src/main/java/wooteco/subway/domain/SubwaySectionsGraph.java

```diff
@@ -0,0 +1,56 @@
+package wooteco.subway.domain;
+
+import java.util.List;
+import java.util.Objects;
+import org.jgrapht.GraphPath;
+import org.jgrapht.alg.shortestpath.DijkstraShortestPath;
+import org.jgrapht.graph.DefaultWeightedEdge;
+import org.jgrapht.graph.WeightedMultigraph;
+
+public class SubwaySectionsGraph {
+
+    private static final String UNREGISTERED_SECTION_STATION = "노선에 등록되지 않은 지하철역입니다.";
+    private static final String NOT_EXIST_PATH = "경로가 존재하지 않습니다.";
+
+    private final WeightedMultigraph<Station, DefaultWeightedEdge> graph;
+
+    public SubwaySectionsGraph(List<Section> sections) {
+        graph = new WeightedMultigraph(DefaultWeightedEdge.class);
```

Comment:
```suggestion
        graph = new WeightedMultigraph<>(DefaultWeightedEdge.class);
```
RawType을 사용하는 경우와 Type을 명시적으로 선언해주는 경우는 어떤 차이가 있을까요?

Reply:
RawType의 경우 컴파일 시점에 타입 에러를 잡아줄 수 없다고 하네요! 명시하도록 수정하였습니다🙇‍♂️